.github/workflows/generate-data.yaml

```diff
@@ -88,214 +95,186 @@ jobs:
           repository: ${{ steps.id.outputs.repo }}
           sparse-checkout-cone-mode: false
           sparse-checkout: |
+            predtimechart-config.yml
             predevals-config.yml
             site-config.yml
-      - id: status-check
+          path: 'cfg/'
+      - id: get-hub-name
+        name: "Get information about hub"
+        run: |
+          yq eval '.hub' cfg/site-config.yml \
+          | sed -E -e 's+(https://github.com/|^)+hub=+' \
+          | sed -E -e 's+[/]$++' >> $GITHUB_OUTPUT
+      - id: status
         name: "Check if we need to build data"
         run: |
-          if [[ ! -f predevals-config.yml ]]; then
-            echo "ok=false" >> "$GITHUB_OUTPUT"
-            repo=${{ steps.id.outputs.repo }}
-            echo "::notice title=Missing config ($repo)::No predevals-config.yml file found. Exiting."
-            exit 0
+          repo=${{ steps.id.outputs.repo }}
+          if [[ ! -f cfg/predevals-config.yml ]]; then
+            echo "eval-ok=false" >> "$GITHUB_OUTPUT"
+            echo "::notice title=$repo: No evals to build::No predevals-config.yml file found."
+          else
+            echo "eval-ok=true" >> $GITHUB_OUTPUT
```

Comment:
this and the `forecast-ok` are indicators based on whether or not the config files exist and they control the downstream jobs.